docker-compose.test.yml

```diff
@@ -33,6 +33,10 @@ services:
       - scrapyd
       - ftp_server
 
+  functional_arxiv:
+    <<: *service_base
+    command: py.test -vv tests/functional/arxiv/test_arxiv.py
```

Comment:
Trim the links to only the really needed services (ex. celery does not need the ftp_server)